packages/model/src/error.ts

```diff
@@ -2,7 +2,7 @@ import { ErrorObject } from 'ajv';
 
 export type ValidationErrorResponse = {
   error: 'Bad Request';
-  message: 'Validation Error';
+  message: 'Validation error';
```

Comment:
it would be easier to have validation error message const

Reply:
to be fair i would have done but i think we want to change the way we throw these errors (see: https://asaphub.atlassian.net/browse/CRN-777) and i think at that point the const will become obsolete - i would imagine we should just use specific error instances? what do you thnink

Reply:
actually i've made the change :D